runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java

```diff
@@ -150,11 +150,22 @@ default boolean confirmKicks()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "kickWithName",
+			name = "Kick with name",
```

Comment:
```suggestion
			name = "Show kicked player",
```

Or something with more intent

Reply:
Kick player by name? More obvious what it does at a glance

Reply:
This is just adding the kicked players name to the game message. It should reflect that. Your sentence implies you are turning on an option to type a players name in to kick them.